lib/maps/utils.js

```diff
@@ -36,24 +45,32 @@ export const zoomToLevel = (zoom, maxZoom) => {
 }
 
 export const getSiblings = (tile) => {
-  let siblings = []
+  let offsets = []
   const deltax = [-2, -1, 0, 1, 2]
   const deltay = [-1, 0, 1]
   const max = Math.pow(2, tile[2]) - 1
   deltax.map((x) => {
     deltay.map((y) => {
       if (!(deltax === 0 && deltay === 0)) {
-        siblings.push(
-          tileToKey([
-            clip(tile[0] + x, 0, max),
-            clip(tile[1] + y, 0, max),
-            tile[2],
-          ])
-        )
+        offsets.push([tile[0] + x, tile[1] + y, tile[2]])
       }
     })
   })
-  const unique = [...new Set(siblings)]
-  unique.unshift(tileToKey([0, 0, tile[2]]))
-  return unique
+
+  // offsets.unshift([0, 0, tile[2]])
```

Comment:
@freeman-lab it wasn't clear to me why we needed this, so I've commented it out for now

Reply:
ah so that was a potentially useless optimization to make the tile at `[0,0]` first, basically, under the assumption that if we kick off that request first it has a higher probability of coming back first (but not guaranteed!)

Reply:
but if we still wanted that behavior we could add a sort at the end, mine was an especially hacky way to do it 😂